Doc update (#160)
MetroCS/QualityToolsForBlueJ: BuildInformation.md

# Quality Tools for BlueJ (QT4BJ) - Build Information

## Required Java version(s)

> Your JDK version should match the requirements of your version of BlueJ. See the [main BlueJ website](https://www.bluej.org/) or [BlueJ installations](https://www.bluej.org/versions.html) to verify that your JDK version is correct

Reviewer: Please use third-person rather than first-person 
```suggestion
The JDK version should match the requirements of the targeted version of BlueJ. See the [main BlueJ website](https://www.bluej.org/) or [BlueJ installations](https://www.bluej.org/versions.html) to verify the correct JDK version to use.
```

Where is the necessary "JDK version" information at either of the given URLs?  These references need to be specific enough that maintainers get the information claimed as available.

Author: I have changed this statement to 3rd person and added information about the necessary JDKs being packaged with the BlueJ installations in my most recent commit.

## Gradle wrapper usage

> A Gradle wrapper class is supplied with this project. Simply replace all `gradle` commands with the `gradlew` command to use the wrapper class and ensure the correct build settings are used.

Reviewer: What does "all `gradle` commands" refer to?

Isn't `gradlew` _"the recommend way to execute any Gradle build"?_ See [Gradle User Guide - Gradle Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html)

Author: I have re-worded my Gradle statements in my most recent commit
